src/Subaccounts/Subaccounts.js

```diff
@@ -136,22 +136,22 @@ class Subaccounts {
     /**
      * @returns {Page}
      */
-    listFirstPage(pageSize = null) {
-        return this.iterator().firstPage(pageSize);
+    listFirstPage(queryParams = null, pageSize = null) {
+        return this.iterator().firstPage(queryParams, pageSize);
     }
 
     /**
      * @returns {Page}
      */
-    listPageAfter(afterId, pageSize = null) {
-        return this.iterator().pageAfter(afterId, null, pageSize);
+    listPageAfter(afterId, pageSize = null, queryParams = null) {
```

Comment:
Let's make the order of the parameters consistend: in listFirstPage 'queryParams' comes before 'pageSize', but in listPageAfter/Before it's the other way around

Reply:
Yes I agree, it would be better to do so, but would't it break the existing code for users?

Reply:
Nvm, I need to change listFirstPage